rotkehlchen/db/upgrades/v39_v40.py

```diff
@@ -203,6 +211,30 @@ def _migrate_ledger_actions(write_cursor: 'DBCursor', conn: 'DBConnection') -> N
                 event_subtype,
             ))
 
+        # copy the taxable ledger actions rules
+        cursor.execute('SELECT value FROM settings WHERE name=?', ('taxable_ledger_actions',))
```

Comment:
move this and the write_cursor for the new accounting rules together under the same if. Why have them separate and why write `new_accounting_rules` if it's just an empty list? Just put it under the if here.

Also just for cleanliness separate into a new upgrade step please.